### DISCUSSION TITLE
How do I remove the limit of three devices on the Dropbox?

### DISCUSSION THREAD

**Discussion by boorgarry** [samschott/maestral#485] [General]
how do you get the three device limit increased for basic Dropbox accounts? it doesn't work for me, what do i have to do, (1iMac, 1iPad pro, 2x iPhon)

**Comment by samschott** [comment]
Not sure what exactly is not working. Maestral is an external app and therefore does not count towards the 3 device limit. However, the limit still applies when using the official client.

**Comment by boorgarry** [comment]
Hi,
ok, I installed Maestral, deleted the original Dropbox folder, now I have Dropbox (Maestral), I'm logged in to iMac, iPad Pro, iPhone, now I want to log into my second iPhone that doesn't work, what am I doing wrong?

Gerhard Bauer
***@***.***

**Reply by boorgarry** [reply]
How do I register with a 4.5, ... device at Dropbox with original access data, it does not work
